Big-Spender/readme.md

```diff
@@ -124,7 +124,17 @@ CREATE YOUR QUERY HERE
 **You:** Sure thing. To confirm, the date is August 19, 2021, the transaction number is 38104091, the supplier invoice number is 3780119655, the supplier is 'Dell', the expense type is 'Hardware' and the expense area is 'IT'. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERIES HERE
+INSERT YOUR QUERIES HERE : INSERT INTO expense_types(expense_type) VALUES('Hardware');
```

Comment:
Sure!

I wonder, if Claire and Farnoosh try this query again on their own, what might happen? Is there a way to protect them from creating duplicate records by mistake?